zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -430,11 +429,52 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
                     cb.call_context_lookup(true.expr(), None, field_tag, 0.expr());
                 }
             },
+        ); // rwc_delta += 3 for empty account
+
+        // handle calls for precompile.
+        cb.condition(
+            and::expr([is_precheck_ok.expr(), is_precompile.expr()]),
```

Comment:
There is already such a condition above, where we do the copy table lookups. We should move these checks also there (so we have only one conditional gate for precompile calls.